app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,44 @@
     <%= javascript_importmap_tags %>
   </head>
 
-  <body>   
-    <% if flash[:alert].present? %>
-        <p class="my_alert"><%= flash[:alert] %></p>
-    <% end %>
-    <%= yield %>
+  <body>  
+    <div class='container-fluid'> 
+      <div class="mt-4 p-5 text-white rounded banner-background">
+        <h1>Customers and Orders</h1>
+        <p>This is a sample application that association orders with customers.</p>
+      </div>
+
+      <% if flash[:notice].present? %>
+        <p style="color: green"><%= flash[:notice] %></p>
```

Comment:
The lesson unfortunately didn't make this clear, but you're supposed to use Bootstrap's [alert classes](https://getbootstrap.com/docs/5.0/components/alerts/) for this.

```suggestion
        <div class="alert alert-success" style="color: green"><%= flash[:notice] %></div>
```

There also doesn't appear to be anything for `flash[:alert]` anymore, so the user will no longer see flash message alerts.